components/perf-tools/msr-safe/SPECS/msr-safe.spec

```diff
@@ -0,0 +1,133 @@
+#---------------------------------------------------------------------------bh-
+# This RPM .spec file is part of the OpenHPC project.
+#
+# It may have been modified from the default version supplied by the underlying
+# release package (if available) in order to apply patches, perform customized
+# build/install configurations, and supply additional files to support
+# desired integration conventions.
+#
+#----------------------------------------------------------------------------eh-
+
+# msr-safe.spec
+
+%include %{_sourcedir}/OHPC_macros
+%{!?PROJ_DELIM: %global PROJ_DELIM -ohpc}
+
+# Base package name
+%define pname msr-safe
+%define PNAME MSR-SAFE
+
+
+Name:           msr-safe
+Version:        1.2.0
+Release:        1
+License:        GPLv3+
+Summary:        Allows safer access to model specific registers (MSRs)
+Url:            https://github.com/LLNL/msr-safe
+Group:          System Environment/Daemons
+Source0:        https://github.com/LLNL/%{name}/archive/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
+Source1:        msr-safe.service
+Source2:        msr-safe.sysconfig
+Source3:        10-msr-safe.rules
+Source4:        msr-safe.sh
+Source5:        OHPC_macros
+Patch0:         0001-Make-msr_batch_op-struct-wmask-field-an-input-parame.patch
+DocDir:         %{OHPC_PUB}/doc/contrib
+Prefix:         %{_prefix}
+BuildRoot:      %{_tmppath}/%{name}-%{version}-build
+BuildRequires:  %kernel_module_package_buildreqs
+BuildRequires:  systemd
+%if 0%{?sles_version} || 0%{?suse_version}
+BuildRequires:  udev
+#!BuildIgnore: post-build-checks
+%endif
+
+%kernel_module_package default
+
+%description
+Allows safer access to model specific registers (MSRs)
+
+%prep
+%autosetup -n %{name}-%{version}
+
+%build
+for flavor in %flavors_to_build; do
+    rm -rf obj/$flavor
+    mkdir -p obj/$flavor
+    cp -r msr* Makefile obj/$flavor
+    %{__make} -C %{kernel_source $flavor} M=$PWD/obj/$flavor
+done
+%{__make} CPPFLAGS="-DVERSION=\\\"%{version}-%{release}\\\"" msrsave/msrsave
+
+%install
+%{__make} install DESTDIR=%{buildroot} prefix=%{_prefix} sbindir=%{_sbindir} mandir=%{_mandir}
+install -d %{buildroot}/%{_datadir}/msr-safe/whitelists
+install -m 0644 whitelists/* %{buildroot}/%{_datadir}/msr-safe/whitelists/
+install -d %{buildroot}%{_unitdir}
+install -m 0644 %{SOURCE1} %{buildroot}%{_unitdir}/msr-safe.service
+install -d %{buildroot}/%{_sysconfdir}/sysconfig
+install -m 0644 %{SOURCE2} %{buildroot}/%{_sysconfdir}/sysconfig/msr-safe
+install -d %{buildroot}/%{_udevrulesdir}
+install -m 0644 %{SOURCE3} %{buildroot}/%{_udevrulesdir}/10-msr-safe.rules
+install -d %{buildroot}%{_sbindir}
+install -m 0755 %{SOURCE4} %{buildroot}%{_sbindir}/msr-safe
+
+export INSTALL_MOD_PATH=$RPM_BUILD_ROOT
+export INSTALL_MOD_DIR=extra/%{name}
+for flavor in %flavors_to_build ; do
+        make -C %{kernel_source $flavor} modules_install \
+                M=$PWD/obj/$flavor
+done
+
+%clean
+rm -rf %{buildroot}
```

Comment:
clean section are automatically included

Reply:
Just to be clear, you are saying we should remove lines 82 and 83.

Reply:
Yes, the %clean section is not needed in SPEC files for quite some time now.

Reply:
Done. Will be in next pull request rev.